src/test/java/core/basesyntax/CsvFileReaderServiceTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.serviceimpl.CsvFileReaderServiceImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CsvFileReaderServiceTest {
+    private static CsvFileReaderServiceImpl csvFileReaderService;
+
+    @Before
+    public void setUp() {
+        csvFileReaderService = new CsvFileReaderServiceImpl();
+    }
+
+    @Test (expected = RuntimeException.class)
+    public void csvFileReader_FilePath_Empty() {
```

Comment:
Use naming approach from 'registration-validation' home work
`<methodUnderTest>_<state>_<expectedBehavior>`
Fix methods names in other places
```suggestion
    public void readDataFromFileCsv_emptyFilePath_notOk() {
```